docs/snapshotting/snapshot-support.md

```diff
@@ -38,7 +36,23 @@ This has the advantage of very fast snapshot loading times, but comes with the c
 of having to keep the guest memory file around for the entire lifetime of the
 resumed microVM.
 
-*Note*: Snapshotting is currently supported only on `x86_64` machines.
+*Note*: Snapshotting is currently supported on Intel and AMD Zen `x86_64` CPUs.
+
+## Functionality and performance
+The current Firecracker snapshot tests and support are targeting only Intel Skylake, Cascade Lake and AMD Zen2 CPUs. These are the officially supported CPUs, but the functionality *may* work on others as well. Our plan is to incrementally extend support for other platforms and CPUs.
+
+Supported host kernel versions: Linux 4.14/4.19. Using Firecracker snapshotting with newer kernels is very likely to function correctly but we do not officially support them yet.
+
+The Firecracker snapshot create/resume performance depends on the memory size,
+vCPU count and emulated devices count. The current CI baseline for snapshot resume latency is under **8ms** with 5ms p90 for a microvm with this specs: 2vCPU/512MB/1 block/1 net device.
```

Comment:
When snapshot tests are run on a m5d.metal instance ($description) for a microvm with we target ....

Reply:
good callout, forgot about setup details.